src/components/CardContainer/CardContainer.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+import { Box, Grid } from "@mui/material";
+import { FilmCard } from "./FilmCard/FilmCard";
```

Comment:
import from folder root as you have added index.ts

Reply:
done